### ISSUE TITLE
Kubernetes Operator for Dataverse

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse#10337]
Operator ideas: https://docs.google.com/document/d/1H7pEBo7nnLsqc23WRIR-BJ5mYf1ym5Dh4Qylc0yXMhY/edit?usp=sharing

**Comment by cmbz** [comment]
2024/07/10
- Any updates @pdurbin ?

**Comment by pdurbin** [comment]
I'd have to go through recent recordings at https://ct.gdcc.io but I believe the status of this is that we (the Containerization Working Group) feel that tagged images (#10618) is a higher priority so the Kubernetes operator is on the back burner.

In addition, we may want to work through some of the milestones we had established in our original [proposal](https://docs.google.com/document/d/14DHDB24Cp_kzpYqhHCKCtnzOw8_WuLOOONyqJHSsaYM/edit) when we established the group. The remaining tasks are now in our [column](https://github.com/orgs/IQSS/projects/34/views/17) and are fairly near the top and have milestones in the titles. Once we complete items from the proposal we can retire the proposal.

Currently, for BacklogClassification this issue is "Prioritized", which is fine, but I'm also ok moving it to "Containerization (Oliver)" so we (the group) can decide when we'd like the tackle it. That is, we could figure out how high in our column it belongs.

I added this to the [agenda](https://docs.google.com/document/d/1mdziHDJTIZGgI1ks8HFNTkfEfQqfsIRmNh4ufF516pk/edit?usp=sharing) of our next meeting (next week) so we can talk about it.

**Comment by pdurbin** [comment]
Oh, one more important update is that back on 2024-03-21 @computate (Christopher Tate from Red Hat) joined the Containerization Working Group call and gave us an excellent brain dump of his experience writing operators (and a few Solr tips as well!). I'm sure when we dive back in to writing an operator we'll review the [notes](https://docs.google.com/document/d/1X69Xig8vgFAMWIS0xw1FIYLT2BgrwImUOWfCt0mwYTw/edit?usp=sharing) and [recording](https://harvard.zoom.us/rec/share/zBqcRUfqUOJYPLXd_orU4rY2oqmIXv8mOnNRAyS-rIBJuV6Vqep6N3O3hRHSRuen.IyVubl9breyMGi8R) of that meeting. Maybe we'll even try to get him back. 😄

**Comment by computate** [comment]
Yeah! Reach back out to Red Hat Research when you're ready to get started.

**Comment by cmbz** [comment]
@pdurbin @poikilotherm Just assigned you to watch this issue, thanks!

**Comment by pdurbin** [comment]
@computate we definitely will! Thanks!

@cmbz no worries. Watching. 👀 

At sprint planning today @cmbz asked us to update any issues we're assigned to that are on hold. The status remains the same. Tagging our images is a higher priority and @poikilotherm has been working away in this PR:

- #10827

We're discussing it here: https://dataverse.zulipchat.com/#narrow/stream/375812-containers/topic/change.20version.20scheme.20base.20image.3F/near/465274491

That's for the base image. After that, we'll work on the application image:

- #10618

After all that, we'll talk about getting back to the Kubernetes operator some day! 😅
